### PR TITLE
Join release dates and support versions to performance events

### PR DESCRIPTION
**Pull request by jwilander**
#### Summary
Need this data to create charts for performance based on version.

### CODE REVIEW

**Comment by jwilander** [comment]
@jasonblais or @nickmisasi if you guys know someone better to review please add them

**Comment by jwilander** [comment]
Also does anyone know how this gets deployed? Is it automated?

File: models/data_warehouse.model.lkml [hunk 1]
```diff
@@ -3868,6 +3868,13 @@ explore: performance_events {
     fields: [server_daily_details.database_version, server_daily_details.database_version_major, server_daily_details.database_version_major_release, server_daily_details.server_version_major, server_daily_details.version, server_daily_details.edition]
   }
 
+  join: version_release_dates {
+    view_label: " Performance Events"
```
Comment (catalintomai) [inline models/data_warehouse.model.lkml:3872]: Is this the correct label?

Comment (jwilander) [reply models/data_warehouse.model.lkml:3872]: I think so based on the pattern already applied

Comment (catalintomai) [reply models/data_warehouse.model.lkml:3872]: asking because the `server_fact` below, for example, uses a different one (does not inherit the parent one)

Comment (jwilander) [reply models/data_warehouse.model.lkml:3872]: Honestly, I'm not really sure either way but it seems to match how `join: server_daily_details` and `join: user_agent_registry` and there's a number of other explores that use the same labels. 0/5, happy to change it if there's a better suggestion.

Comment (sri-byte) [reply models/data_warehouse.model.lkml:3872]: yeah, the labels are incorrect for `join: server_daily_details` and `join: user_agent_registry`, @jwilander please rename the view label to 'Version Release Dates'. and leave the old ones as is, I will fix them soon.

Comment (jwilander) [reply models/data_warehouse.model.lkml:3872]: Done @sri-byte @catalintomai

**Review by catalintomai — APPROVED** [review]

File: models/data_warehouse.model.lkml (hunk 1 above)
Comment (catalintomai) [reply models/data_warehouse.model.lkml]: @sri-byte , when you do the name fixing, please open a Jira ticket for the task to keep track (also would be a good idea to do see if there are some other enhancements as part of the scrub, e.g. adding descriptions, removing unused joins, etc.) 0/5 if you want to open separate Jira tickets by improvement category.

**Comment by jwilander** [comment]
@sri-byte any concern if I merge?

**Comment by sri-byte** [comment]
All good @jwilander. Please merge.

**Pull request merged**